notebooks/Specviz2dExample.ipynb

```diff
@@ -61,8 +73,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from astropy.utils.data import download_file\n",
-    "fn = download_file('https://stsci.box.com/shared/static/exnkul627fcuhy5akf2gswytud5tazmw.fits', cache=True)"
+    "data_dir = tempfile.gettempdir()\n",
+    "#data_dir = \"/User/username/Data/\"\n",
+    "\n",
+    "fn = \"jw01538-o160_s00004_nirspec_f170lp-g235h-s1600a1-sub2048_s2d.fits\"\n",
+    "uri = f\"mast:JWST/product/{fn}\"\n",
+    "result = Observations.download_file(uri, f'{data_dir}/{fn}')"
```

Comment:
For some reason without this, I get a traceback

```suggestion
    "result = Observations.download_file(uri, local_path=f'{data_dir}/{fn}')"
```

Reply:
In case you're curious

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In [6], line 6
      4 fn = "jw01538-o160_s00004_nirspec_f170lp-g235h-s1600a1-sub2048_s2d.fits"
      5 uri = f"mast:JWST/product/{fn}"
----> 6 result = Observations.download_file(uri, f'{data_dir}/{fn}')

TypeError: ObservationsClass.download_file() takes 2 positional arguments but 3 were given
```

Reply:
Whoops, didn't mean to delete the arg name. One sec.